src/Libraries/CoreNodeModels/Input/FileSystem.cs

```diff
@@ -259,7 +259,7 @@ private static IEnumerable<TItem> Singleton<TItem>(TItem x)
         }
     }
 
-    [NodeName("File.FromPath")]
+    [NodeName("File From Path")]
```

Comment:
so does this node show as File From Path or FileSystem. File From Path?

Reply:
It shows as `File From Path`.  @Racel requested nodes that can be called from DesignScript use their expected syntax while nodes that cannot (nodeModel) be named as shown above.

![image](https://user-images.githubusercontent.com/13341935/32730062-e01f58a2-c853-11e7-9224-00a831d71de7.png)

Reply:
@alfarok @Racel @QilongTang I don't understand why this decision was made...
<img width="819" alt="screen shot 2017-11-13 at 10 07 55 am" src="https://user-images.githubusercontent.com/508936/32732451-9c7f2a8a-c85a-11e7-99da-42f43b1aa81d.png">

you can call these nodes from codeblocks - they just don't appear to show in autocomplete.

Reply:
I was a bit unsure on the naming difference as well but I didn't know you could still call them from designScript even if they don't appear in autocomplete.  @Racel can we switch these back to `FileSystem.FileFromPath` as opposed to `File From Path` 

![image](https://user-images.githubusercontent.com/13341935/32733289-dc798732-c85c-11e7-8493-b649194e2f11.png)

Reply:
@alfarok @mjkkirschner I wasn't aware the decision was changed, description of Jira Task still shows `FileFromPath` without space in between

Reply:
@QilongTang @mjkkirschner @Racel Did a bit of digging into this and I don't think it is possible to call NodeModel nodes from DesignScript.  The reason it worked in the example Mike posted above is because there is actually a hidden ZeroTouch node that gets call by the NodeModel node.  Hidden nodes can be accessed from DesignScript so you are actually calling the ZeroTouch representation which is wrong.  We believe the reason the node was made into a NodeModel node was to force the whole graph to re-execute when the file path is changed (harder to accomplish with ZT).  This being said I think the current format is actually correct.

Reply:
You may try and add a migration here from "File.FromPath" to "File From Path": https://github.com/DynamoDS/Dynamo/pull/8338/files#diff-0f7073fd4c02eb5916842001d840c964L269 like:
```[AlsoKnownAs("DSCore.File.FileObject", "DSCoreNodesUI.Input.FileObject", "File.FromPath")]
```
and see if that works.